### PR TITLE
Enable Preview plan in KEB

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- create Preview plan
- does not bump the image, this PR is needed for next Lifecycle Manager integration tasks

**Related issue(s)**

### CODE REVIEW

File: components/kyma-environment-broker/cmd/broker/provisioning_test.go [hunk 1]
```diff
@@ -46,6 +46,34 @@ func TestProvisioning_HappyPath(t *testing.T) {
...
+					"parameters": {
+						"name": "testing-cluster"
+					}
+		}`)
+	opID := suite.DecodeOperationID(resp)
+
+	suite.processProvisioningByOperationID(opID)
+
+	suite.WaitForOperationState(opID, domain.Succeeded)
+	// todo: assert Kyma resource is created
```
Comment (wozniakjan) [inline components/kyma-environment-broker/cmd/broker/provisioning_test.go:74]: nit: is it planned as part of this PR or a later follow-up?

File: components/kyma-environment-broker/cmd/broker/main.go [hunk 2]
```diff
@@ -666,6 +666,8 @@ func NewProvisioningProcessingQueue(ctx context.Context, provisionManager *proce
 		{
 			stage: createRuntimeStageName,
 			step:  provisioning.NewOverridesFromSecretsAndConfigStep(db.Operations(), runtimeOverrides, runtimeVerConfigurator),
+			// Preview plan does not calls Reconciler so it does not need overrides
```
Comment (wozniakjan) [inline components/kyma-environment-broker/cmd/broker/main.go:669]: nit: in that case, I think we can skip BTP Operator related steps as well here and in the `update` queue.

**Review by wozniakjan — COMMENTED** [review]
if the `update` operations are allowed for preview plans, we may want to skip reconciler interactions there too

File: components/kyma-environment-broker/cmd/broker/provisioning_test.go (hunk 1 above)
Comment (piotrmiskiewicz) [reply components/kyma-environment-broker/cmd/broker/provisioning_test.go:74]: it is for later PR when create kyma steop is done

File: components/kyma-environment-broker/cmd/broker/main.go (hunk 2 above)
Comment (piotrmiskiewicz) [reply components/kyma-environment-broker/cmd/broker/main.go:669]: BTP operator steps must be run - we will provide thhe secret directly to SKR

Comment (wozniakjan) [reply components/kyma-environment-broker/cmd/broker/main.go:669]: and just to make sure, is it ok to run these reconciler interactions in the update queue? or do we want to also skip them?
https://github.com/kyma-project/control-plane/blob/e3b5f1eab946516db36765a7bf8a56c81430f0e5/components/kyma-environment-broker/cmd/broker/main.go#L767-L774

Comment (wozniakjan) [reply components/kyma-environment-broker/cmd/broker/main.go:669]: for the record, @piotrmiskiewicz just explained to me it's actually skipped for preview plan, just [here](https://github.com/kyma-project/control-plane/pull/2179/files#diff-84e967d32a4a3ed65022482cc8724cfd81b0b8e902c2a835775a40e009e04b49R10), I completely missed that

**Review by wozniakjan — APPROVED** [review]

**Pull request merged**